tasks/25_rot13-encoder/rot13-encoder.spec.js

```diff
@@ -13,7 +13,7 @@ xdescribe('rot13-encoder', () => {
         expect(rot13Encoder("LOVE?")).toBe('YBIR?');
     });
 
-    test('rot13Encoder("Hello World!") should encode to "Uryyb Jbeyq!"', () => {
-        expect(rot13Encoder("LOVE?")).toBe('YBIR?');
+    test('rot13Encoder("HELLO WORLD!") should encode to "URYYB JBEYQ!"', () => {
+        expect(rot13Encoder("HELLO WORLD!")).toBe("URYYB JBEYQ!");
```

Comment:
yeah there is a mistake, but function should work with small letters as well: 
```js
 expect(rot13Encoder("Hello World!")).toBe('Uryyb Jbeyq!');
```

Reply:
Ok, but in this case please update README for this task because now it clearly says that "All letters will be uppercase."

Reply:
ah, sorry, my mistake. Leave it as is then